astropy/uncertainty/function_helpers.py

```diff
@@ -151,7 +151,7 @@ def concatenate(arrays, axis=0, out=None, dtype=None, casting="same_kind"):
 # Add any dispatched or helper function that has a docstring to __all__, so
 # they will be typeset by sphinx. The logic is that for those presumably the
 # way distributions are dealt with is not entirely obvious.
-__all__ += sorted(
+__all__ += sorted(  # noqa: PLE0605
```

Comment:
I think this is a case where the PR fixing all `__all__` was reviewed at the same time that my PR introduced an `__all__` in `uncertainty`. Sometimes, parallel work fails!  Thanks for fixing, @nstarman!